manager/gui/src/components/alerts/LrAlert.vue

```diff
@@ -25,8 +25,11 @@
       :multiSelectButtons="alertConst.alertTable.multiSelectButtons"
```

Comment:
Other table design changes, column width and checkboxes.

Reply:
Done